tests/suites/os/tests/device-tree/index.js

```diff
@@ -213,7 +214,31 @@ module.exports = {
 					targetState.local.config.HOST_CONFIG_dtparam,
 					'DTparams successfully configured in config.txt',
 				);
+				/* Static binary currently shared by RPI maintainer in gdrive only
+				 * See: https://github.com/raspberrypi/Raspberry-Pi-OS-64bit/issues/67#issuecomment-653209729
+				 */
+				test.is(
+					await this.context
+						.get()
+						.worker.executeCommandInHostOS(
+							'cd /tmp/ && curl -L "https://drive.google.com/uc?export=download&id=1HS9E5vnxxNqrizB4mEYrnFoQQ1axSRKm" -o vcdbg && chmod +x ./vcdbg && \
```

Comment:
Would it be worth adding this binary to git just so we never lose it? I'm not sure I trust that drive link to be around forever.

Reply:
Okay, as long as we are prepared for this test to break when they remove the drive link and distribute it properly. Maybe that will be a good forcing function for us!